versions/ufo3/glyphs/glif.md

```diff
@@ -315,14 +315,18 @@ The following is a registry of public lib keys that map to functionality that is
 
 This key is used for representing the "mark" color seen in various font editors. The value for the key must be a string following the [color definition] standard. This data is optional.
 
+##### public.postscriptName
+
+This key is used for representing the indended "production" name for the glyph, which in OpenType fonts would be typically stored in the `CFF ` or `post` table. The value for the key must be a string. It is recommended that the value conforms to the [Adobe Glyph List Specification](https://github.com/adobe-type-tools/agl-specification). Authoring tools may prefer this value over the `glyph` element's `name` attribute when converting the UFO into an OpenType font, but it is solely up to the authoring tool whether it would expect the data in [features.fea](../features.fea) to correspond to the the `glyph` element's `name` attribute or the `public.postscriptName` value. This data is optional. 
```

Comment:
Maybe "solely" is too much. I was trying to say that there is a potential uncertainty but it's out of scope of the spec to mandate any behavior. That clarification may not be really needed but I prefer to write specs so that they explicitly say not just what's covered but also what's not covered by design.